code/processes/tickerlogreplay.q

```diff
@@ -28,6 +28,7 @@ partandmerge:@[value;`partandmerge;0b];                                 // setti
 tempdir:@[value;`tempdir;`:tempmergedir];                               // location to save data for partandmerge replay
 mergenumrows:@[value;`mergenumrows;10000000];                           // default number of rows for merge process
 mergenumtab:@[value;`mergenumtab;`quote`trade!10000 50000];             // specify number of rows per table for merge process
+zipped:@[value;`zipped;0b];                                             // whether in zipped mode or not
```

Comment:
think this comment could be a bit more descriptive, maybe something like

`// enable support for gzipped log files`